x-pack/legacy/plugins/ml/public/application/components/chart_tooltip/chart_tooltip.tsx

```diff
@@ -7,10 +7,7 @@
 import classNames from 'classnames';
 import React, { useRef, FC } from 'react';
 import { TooltipValueFormatter } from '@elastic/charts';
-
-// TODO: Below import is temporary, use `react-use` lib instead.
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { useObservable } from '../../../../../../../../src/plugins/kibana_react/public/util/use_observable';
+import useObservable from 'react-use/lib/useObservable';
```

Comment:
Could we make this `import {useObservable} from 'react-use';` like it's done in react-use's docs? https://github.com/streamich/react-use/blob/master/docs/useObservable.md

Reply:
Imports of specific hooks is also documented: https://github.com/streamich/react-use/blob/master/docs/Usage.md#usage

I thought, that we don't have tree shaking and if we indeed don't have them, then makes sense to leave those specific imports.

Reply:
Fair enough! LGTM